packages/model-viewer/src/features/environment.ts

```diff
@@ -168,7 +233,7 @@ export const EnvironmentMixin = <T extends Constructor<ModelViewerElementBase>>(
         return;
       }
       this[$currentEnvironmentMap] = environmentMap;
-      this[$scene].environment = this[$currentEnvironmentMap];
+      // this[$scene].environment = this[$currentEnvironmentMap];
```

Comment:
I've temporarily disabled this to show how Three.js handles a model with no environment map to define specular reflections.